### ISSUE TITLE
Stacking Enhancement bonuses and Natural Attacks

### ISSUE THREAD

**Issue by semiliterate** [Vek17/WrathMods-TabletopTweaks#246]
Magus and Warpriest classes can add enhancement bonuses that stack with existing enhancement bonuses up to +5. There are several ways to get an enhancement bonus to an unarmed strike, such as Amulet of Mighty Fists or the Power Fist. However, these enhancement bonuses interact weirdly in two ways.

Firstly, imparted bonuses to hit fail to stack when they explicitly should; they are possibly applied in the wrong order. Magus and Warpriest abilities increase the base enhancement bonus of the actual weapon being held, such that the fist or claw item now appears with a +1 or higher bonus listed in its properties. Other abilities such as Silky Veil, Amulet of Mighty Fists, and Power Fist, apply the bonus directly to attack roll calculations. If the implementation was vice versa, or item bonuses were imparted to the active weapon, they would correctly stack. Stacking is most likely RAW - in the absence of FAQ, the effect of Amulet of Mighty Fists +1 is "an existing enhancement bonus" to your unarmed strikes.

Secondly, damage bonuses from these enhancement bonuses stack endlessly. A Magus with a +1 class ability enhancement and a +1 amulet of mighty fists gets +2 to damage, which is correct. However, wearing Power Fist, Silky Veil etc in addition adds further damage bonuses. RAW, these items are enhancement bonuses to damage from different sources, and should not stack; only the Magus and Warpriest abilities (and Paladin Divine Weapon bond etc) explicitly say they stack with existing enhancement bonuses.

To summarise: From these sources, Magus ability enhancement to hit and damage should stack with item enhancement bonuses. Item enhancement bonuses to hit and damage should not stack with each other. Currently all enhancement bonuses to damage from this list stack, and no enhancement bonuses to hit stack.
